python/mxnet/ndarray/sparse.py

```diff
@@ -1151,6 +1177,284 @@ def _ndarray_cls(handle, writable=True, stype=_STORAGE_TYPE_UNDEFINED):
 _set_ndarray_class(_ndarray_cls)
 
 
+def add(lhs, rhs):
```

Comment:
Need to update the document and clarify that it's equivalent to nd.elemwise_add() when shape matches. Same for other mul, div, sub

Reply:
Done